src/main/java/uz/bookstore/bookstore/controller/CartControllerImpl.java

```diff
@@ -0,0 +1,51 @@
+package uz.bookstore.bookstore.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uz.bookstore.bookstore.dto.CartDTO;
+import uz.bookstore.bookstore.entity.Book;
+import uz.bookstore.bookstore.service.CartService;
+
+import java.util.List;
+
+@RequestMapping("/api/v1/cart")
+@RestController
+@RequiredArgsConstructor
+public class CartControllerImpl {
+    private final CartService cartService;
+
+    @PostMapping("/{id}")
+    public ResponseEntity<?> addBookToCart(@PathVariable Long id, @RequestBody Book book) {
+        CartDTO result = cartService.addBookToCart(id, book);
+        return ResponseEntity.status(HttpStatus.ACCEPTED).body(result);
```

Comment:
We only return ResponseEntity.ok we do not need statuses like ACCEPTED or something else.